credentialsdemo/source/no-sdk/src/org/kaaproject/kaa/examples/credentials/utils/IOUtils.java

```diff
@@ -0,0 +1,230 @@
+/**
+ *  Copyright 2014-2016 CyberVision, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.kaaproject.kaa.examples.credentials.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.security.InvalidKeyException;
+import java.security.KeyFactory;
+import java.security.KeyPair;
+import java.security.KeyPairGenerator;
+import java.security.NoSuchAlgorithmException;
+import java.security.PrivateKey;
+import java.security.PublicKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.PKCS8EncodedKeySpec;
+import java.security.spec.X509EncodedKeySpec;
+import java.util.regex.Pattern;
+
+public class IOUtils {
+
+    public static String SANDBOX_IP_ADDRESS = "provided by user";
+    public static final int DEFAULT_KAA_PORT = 8080;
+
+    public static final String PRIVATE_KEY_LOCATION = "key.private";
+    public static final String PUBLIC_KEY_LOCATION = "key.public";
+
+    public static String TENANT_ADMIN_USERNAME = "admin";
+    public static String TENANT_ADMIN_PASSWORD = "admin123";
+
+    public final static String APPLICATION_NAME = "Credentials demo";
```

Comment:
Leave it in IOUtils because of they only save some value and don't matter for application logic. If they will be changed, logic doesn't vary